webtask.json

```diff
@@ -1,8 +1,8 @@
 {
   "title": "Auth0 Authorization",
   "name": "auth0-authz",
-  "version": "2.6.5",
-  "preVersion": "2.5.2",
+  "version": "2.7.0",
+  "preVersion": "2.6.5",
```

Comment:
What does the preVersion do again? I don't recall having to change this in the other extension.

Reply:
I am not really sure. I've checked with DAE and didn't see it there either. I'll get rid of it.

Reply:
https://github.com/auth0/auth0-authorization-extension/pull/260/commits/a4b5fa0c5233cdab72b48c589ed62cd2015848eb